documentation/pe_puppetdb_notes.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: "PuppetDB 3.2 » PE PuppetDB Notes"
+layout: default
+canonical: "/pe-puppetdb-notes"
+---
+
+[reports]: ./api/query/v4/reports.html
+[terminus]: ./puppetdb_connection.markdown#include_unchanged_resources
+
+## 2015.3.0
```

Comment:
Hmmm .... a user has already navigated to PDB 3.2 somehow, which yes applies to 2015.3.0. I dunno, this feels like release notes to me, but PE specific ... is a different file really the correct way to do this? I mean, if its chronological, then its release notes ... we can easily add the PE version to release notes perhaps reference PE versions. See what I mean?

If its a list of differences between PE and FOSS, then that makes sense, but then its not as useful chronologically, its more useful as a list of features or differences or something.